app/views/sessions/new.html.erb

```diff
@@ -1,14 +1,52 @@
-<% provide :title, t("log_in") %>
-<h1><%= t("log_in") %></h1>
+<% provide :title, t(:log_in) %>
+<h1><%= t(:course_registration) %></h1>
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
     <%= form_for :session, url: login_path do |f| %>
       <%= f.label :email, t(:email)  %>
       <%= f.email_field :email, class: "form-control" %>
+      <%= f.label :password, t(:password)  %>
       <%= link_to "(forgot password)", password_resets_path %>
       <%= f.password_field :password, class: "form-control" %>
       <%= f.submit "Log in", class: "btn btn-primary" %>
     <% end %>
+
     <p><%= t "new_user?" %><%= link_to t("sign_up_now!"), signup_path %></p>
+
+    <div class="field">
```

Comment:
What is purpose for these code? from line 16 -> 49

Reply:
to allow user to choose several courses that are listed there

Reply:
is it in order to from log in page to course page, i need to use redirect_to @courses in courses_controller?